target/linux/ramips/image/mt7621.mk

```diff
@@ -228,6 +233,21 @@ define Device/dlink_dir-860l-b1
 endef
 TARGET_DEVICES += dlink_dir-860l-b1
 
+define Device/dlink_dir-878-a1
+  IMAGE_SIZE := 16000k
+  DEVICE_VENDOR := D-Link
+  IMAGES += factory.bin
```

Comment:
I'd move this down directly before setting up the images (IMAGE/*) ...

Reply:
reordered